app/posts/find-teacher-training/2021-08-06-changes-to-find-at-the-end-of-2021-cycle.md

```diff
@@ -0,0 +1,114 @@
+
+## Candidates are discouraged from searching for courses after the deadline to apply again has passed
+
+Although candidates cannot submit an application after the [deadline to apply again](/find-teacher-training/changes-to-find-at-the-end-of-2021-cycle/#21-september-2021-at-6pm) pas passed, we continue to show course listings so that providers and candidates who’ve already applied can finalise any post-submission activity (such as making and accepting offers).
+
+A banner lets candidates know that courses are closed but that they can start or continue applying for the next cycle anyway:
+
+![The deadline to apply again has passed.](courses-closed-banner.png)
+
+We also remove the [‘Apply for this course’ button](/find-teacher-training/changes-to-find-at-the-end-of-2021-cycle/#if-a-candidate-applying-for-the-first-time-tries-to-select-a-course-after-the-deadline) after the deadline to apply again because it implies that you're applying for a particular course, when in fact there's no guarantee that the 2021 to 2022 course listings will be available the following cycle.
+
+The following message replaces the button, enabling candidates to start or continue an application despite courses being closed:
+
+![Stop allowing candidates to click 'Apply for this course'.](courses-closed-start-application-anyway.png)
+
+As before, candidates do not go through the ['you selected a course' journey](/find-teacher-training/changes-to-find-at-the-end-of-2021-cycle/select-course-journey.png).
+
+## Candidates can no longer view courses when the recruitment cycle has ended
+
+[When the recruitment cycle ends](/find-teacher-training/changes-to-find-at-the-end-of-2021-cycle/#4-october-2021-at-midnight) candidates can no longer be placed on courses starting in the 2021 to 2022 academic year, so we stop showing the 2021 to 2022 courses.
+
+Anyone trying to visit Find will be informed that courses are closed, but we enable them start or continue an application anyway:
+
+![Course listing are not shown when the recruitment cycle ends.](find-does-not-show-courses-when-the-cycle-has-ended.png)
+
+## Candidates can browse courses when the recruitment cycle begins
+
+When the [recruitment cycle begins](/find-teacher-training/changes-to-find-at-the-end-of-2021-cycle/#5-october-2021-at-9am) candidates can view courses starting in the 2022 to 2023 academic year.
+
+The ‘Apply for this course’ button is reinstated on course pages.
+
+Candidates can add courses to their application but they cannot submit it until [applications open](/find-teacher-training/changes-to-find-at-the-end-of-2021-cycle/#12-october-2021-at-9am).
+
+{% from "govuk/components/inset-text/macro.njk" import govukInsetText %}
+{{ govukInsetText({
+  text: "On clicking 'Apply for this course', users will no longer be asked whether they want to apply on UCAS as it will no longer be a route to apply for teacher training from October 2021."
+}) }}
```

Comment:
Suggest dropping this and including it on a different page about post-UCAS changes?